docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+services:
+
+  mongo:
+    image: mongo:3.2
+    volumes:
+      - ./whydDB:/data # Data Persistance and Inits
+
+  web:
+    build: .
+    volumes:
+      - ./whydJS:/usr/src/app
+      - /usr/src/app/node_modules
+    ports:
+      - "8080:8080"
+    depends_on:
+      - mongo
+    environment:
+      # From whydJS/env-vars-testing.sh not working in Dockerfile
```

Comment:
Yes but docker manage ENV on its own and I didn't succeed. Also you have to keep here at least `MONGODB_HOST=mongo` and not redefining it in script.